src/style.css

```
@@ -1 +1,71 @@
-/* Styles go here */
+body {
+  font-family: "Roboto", sans-serif;
+  box-sizing: border-box;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  height: 60px;
+  background-color: #fff;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+/* .header::after {
+  content: "";
+  margin-right: 25px;
+} */
+
+.logo {
+  height: 40px;
+  width: 40px;
+  display: flex;
+  align-self: center;
+  /*padding-left: 50px;*/
+}
+
+.nav-list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-link {
+  padding: 22.5px auto;
```

```suggestion
  padding: 22.5px auto;
```
Щоб не вказувати такі цифри, можна центрувати за допомогою justify-content: center та align-items: cemter